_sass/blocks/_landing.scss

```diff
@@ -0,0 +1,84 @@
+.landing-section_top,
+.landing-section_open,
+.landing-section {
+  padding-bottom: $base-padding;
+  padding-top: $base-padding;
+}
+
+.landing-section {
+  @include respond-to(medium-up) {
+    @include span-columns(5.5);
+
+    border-bottom: none;
+
+    &:nth-child(even) {
+      @include shift(1);
+    }
+  }
+
+  border-bottom: $thin-border-size solid $mid-gray;
```

Comment:
Expected item on line 19 to appear before line 9. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets